src/platform/user/profile/vet360/components/AddressField/AddressField.jsx

```diff
@@ -105,8 +105,8 @@ function AddressField({ title, fieldName, deleteDisabled }) {
       Content={AddressView}
       EditModal={AddressEditModal}
       ValidationModal={AddressValidationModal}
-      formSchema={getFormSchema(fieldName)}
-      uiSchema={getUiSchema(fieldName)}
+      formSchema={getFormSchema()}
```

Comment:
I am not too familiar with how this works - home come we no longer have to pass `fieldName`?

Reply:
Because that function stopped using args a while ago :)